src/Installer/DockerInstaller.php

```diff
@@ -24,6 +22,6 @@ public function __construct(ChainBuilder $tasks)
      */
     public function install()
     {
-        $this->tasks->getRunner()->run();
+        $this->tasks->getProvider()->getTasks()->getRunner()->run();
```

Comment:
Yes, this certainly isn't good. But to just inject the runner, what builds the runner? Should we have a class that basically doesn't do almost anything for each of the steps? Or should we do that in the container? Doesn't seem great either...